plugins/colored-man-pages/colored-man-pages.plugin.sh

```diff
@@ -0,0 +1,24 @@
+#! bash oh-my-bash.module
+
+# Bold and Blinking
+export LESS_TERMCAP_mb=$_omb_term_green
+export LESS_TERMCAP_md=$_omb_term_bold$_omb_term_green
```

Comment:
```suggestion
export LESS_TERMCAP_md=$_omb_term_bold_green
```
There are actually variables for foreground colors with boldface.